Project.toml

```
@@ -7,7 +7,7 @@ version = "0.2.0"
 SymbolicUtils = "d1185830-fcd6-423d-90d6-eec64667417b"
 
 [compat]
-SymbolicUtils = "1.5.0"
+SymbolicUtils = "2"
```

Any reason to drop support for 1.5?
```suggestion
SymbolicUtils = "1.5.0, 2"
```